frameworks/lightautoml/exec.py

```diff
@@ -51,7 +52,13 @@ def run(dataset, config):
             ]).T
 
         predictions = np.argmax(probabilities, axis=1)
-
+        class_map = automl.outer_pipes[0].ml_algos[0].models[0][0].reader.class_mapping
+        if class_map is None and df_train[label].dtype == bool:
+            class_map = {False: 0, True: 1}
+        if class_map:
+            column_to_class = {col: class_ for class_, col in class_map.items()}
+            predictions = list(map(column_to_class.get, predictions))
+            probabilities_labels = list(class_map)
```

Comment:
If I understand that `col` is always an integer, then the following is the only way to guarantee that `probabilities_labels` will be sorted correctly:
```suggestion
            probabilities_labels = [class_map[k] for k in sorted(class_map)]
```

Reply:
`class_map` is a dictionary from class to column, e.g. `{"b": 0, "a": 1}` sorted in order of columns.
Should `probabilities_labels` not match the order of the columns of `probabilities`? `sorted(class_map)` will just put the labels in alphabetical order. If you mean that you don't want to rely on the order of `class_map` then I suppose you'd want `[column_to_class[col] for col in sorted(column_to_class)]`?

Reply:
> class_map is a dictionary from class to column, e.g. {"b": 0, "a": 1} sorted in order of columns.

is it guaranteed by contract? as hidden implementation detail?

> [column_to_class[col] for col in sorted(column_to_class)]?

yes, that's what I meant, sorry.

Reply:
I can't find `class_mapping` in their documentation at all so I will assume it's an implementation detail and explicitly ensure the order.

Reply:
thanks

Reply:
```suggestion
            probabilities_labels = [column_to_class[col] for col in sorted(column_to_class)]
```